### PR TITLE
Hide L2 assets with price = 0

### PR DESCRIPTION
**Pull request by brunobar79**
We won't hide it if the user has it pinned.

PoW: http://recordit.co/a416SSME1S

### CODE REVIEW

**Review by estebanmino — APPROVED** [review]
👌

**Comment by skylarbarrera** [comment]
wonder if we only want to do this for polygon since its only an issue there, at least currently

File: src/redux/data.js [hunk 1]
```diff
@@ -542,6 +543,14 @@ export const addressAssetsReceived = (
     );
     dispatch(subscribeToMissingPrices(missingPriceAssetAddresses));
   }
+
+  // Hide coins with price = 0 that are currently not pinned
+  if (isL2) {
+    const assetsWithNoPrice = parsedAssets
+      .filter(asset => asset.price?.value === 0)
+      .map(({ address }) => address);
+    dispatch(addCoinsToHiddenList(assetsWithNoPrice));
```
Comment (skylarbarrera) [inline src/redux/data.js:552]: this will re-hide assets after you hide them

**Review by skylarbarrera — APPROVED** [review]
🙆‍♂️

**Pull request merged**